api_auth.gemspec

```diff
@@ -9,18 +9,20 @@ Gem::Specification.new do |s|
   s.authors = ['Mauricio Gomes']
   s.email = 'mauricio@edge14.com'
 
-  s.add_development_dependency 'appraisal'
-  s.add_development_dependency 'rake'
-  s.add_development_dependency 'amatch'
-  s.add_development_dependency 'rspec', '~> 3.4'
+  s.required_ruby_version = '>= 2.1.0'
+
   s.add_development_dependency 'actionpack', '< 6.0', '> 4.0'
+  s.add_development_dependency 'activeresource', '>= 4.0'
```

Comment:
Can we make this `'< 6.0', '> 4.0'` like activesupport?

Reply:
The release cycle of activeresource is not the same has rails (activesupport, actionpack...).
https://rubygems.org/gems/activeresource/versions/5.0.0

I thinks it's a bad idea to block development_dependency because we can not run test with the last version, but someone can use ApiAuth with the last version. See the kind of problem we can have with #152 or #125.

Reply:
I know it's not the same as Rails. 5.0 is the current major version. I'd hate for 6.0 to come out with breaking changes, and a dev working on this gem might start seeing test failures without realizing why.

Though as your point out, this is kind of complicated by the fact that this gem can be used with "untested" versions of libraries due to not wanting to pull in every potential gem when installed into an app.

Reply:
I have do a quick look on Gem::Specification and bundler, there is not possible to define optional_dependency :(

Reply:
Okay, we'll just cross the 6.0 bridge when we come to it then. It's a dev only impact anyway.